resources/idlharness.js

```diff
@@ -2862,10 +2863,11 @@ IdlInterface.prototype.test_interface_of = function(desc, obj, exception, expect
             continue;
         }
         if (!exposed_in(exposure_set(member, this.exposureSet))) {
+            var details = member.type === "operation" ? member.toString() : member.name;
             subsetTestByKey(this.name, test, function() {
                 assert_equals(exception, null, "Unexpected exception when evaluating object");
                 assert_false(member.name in obj);
-            }.bind(this), this.name + " interface: " + desc + ' must not have property "' + member.name + '"');
+            }.bind(this), this.name + " interface: " + desc + ' must not have property "' + details + '"');
```

Comment:
Is this and the next change needed to avoid duplicate test names? Since only the operation (method) name is used, it doesn't really make sense to have a test like 'Navigator interface: navigator must not have property "sendBeacon(USVString, optional BodyInit?)"', which this results in.

If we need to avoid duplicate tests here, we should probably instead keep track of which property names we've already tested for, rather than having multiple tests doing exactly the same thing but with different names.

Reply:
Hi @foolip, actually I verified the first and third changes to avoid duplicate test names. Regarding to this second changes, I just educe it maybe also need modify according to others two modifications.

Reply:
a soft ping @foolip, thanks.

Reply:
OK, so it sounds like we are creating one test for each method overload, but the tests would do exactly the same thing with different names. Is that right? Instead of this, can we add an "already created test for this" set and check that?

Reply:
@foolip I've updated, please take another look, thanks.

Reply:
@foolip gentle ping, PTAL, thanks.